tests/basic.ispec.js

```diff
@@ -4,176 +4,297 @@ import { createHash } from "crypto";
 
-test("showAddressAndPubKey", async () => {
-  jest.setTimeout(60000);
+    // Derivation path. First 3 items are automatically hardened!
+    const path = [44, 474, 5, 0, 3];
+    const resp = await app.showAddressAndPubKey_ed25519(path);
 
-  const transport = await TransportNodeHid.create(1000);
-  const app = new OasisApp(transport);
+    console.log(resp);
 
-  // Derivation path. First 3 items are automatically hardened!
-  const path = [44, 474, 5, 0, 3];
-  const resp = await app.showAddressAndPubKey(path);
+    expect(resp.return_code).toEqual(0x9000);
+    expect(resp.error_message).toEqual("No errors");
 
-  console.log(resp);
+    expect(resp).toHaveProperty("bech32_address");
+    expect(resp).toHaveProperty("pk");
 
-  expect(resp.return_code).toEqual(0x9000);
-  expect(resp.error_message).toEqual("No errors");
+    expect(resp.bech32_address).toEqual("oasis1qphdkldpttpsj2j3l9sde9h26cwpfwqwwuhvruyu");
+    expect(resp.pk.length).toEqual(32);
+  });
 
-  expect(resp).toHaveProperty("bech32_address");
-  expect(resp).toHaveProperty("pk");
+  test("getAddressAndPubKey_Secp256k1", async () => {
+    const app = new OasisApp(transport);
 
-  expect(resp.bech32_address).toEqual("oasis1ectg9qs5f7vlnlxce96gm264tqjng0glydhhnk2h4e4tgn442lgtwzrun");
-  expect(resp.pk.length).toEqual(32);
-});
+    // Derivation path. First 3 items are automatically hardened!
+    const path = [44, 60, 0];
+    const resp = await app.getAddressAndPubKey_secp256k1(path);
 
-test("appInfo", async () => {
-  const transport = await TransportNodeHid.create(1000);
-  const app = new OasisApp(transport);
+    console.log(resp);
 
-  const resp = await app.appInfo();
+    expect(resp.return_code).toEqual(0x9000);
+    expect(resp.error_message).toEqual("No errors");
 
-  console.log(resp);
+    expect(resp).toHaveProperty("hex_address");
+    expect(resp).toHaveProperty("pk");
 
-  expect(resp.return_code).toEqual(0x9000);
-  expect(resp.error_message).toEqual("No errors");
+    expect(resp.hex_address).toEqual("95e5e3c1bdd92cd4a0c14c62480db5867946281d");
+    expect(resp.pk.toString("hex")).toEqual(
+      "021853d93524119eeb31ab0b06f1dcb068f84943bb230dfa10b1292f47af643575",
+    );
+    expect(resp.pk.length).toEqual(33);
+  });
 
-  expect(resp).toHaveProperty("appName");
-  expect(resp).toHaveProperty("appVersion");
-  expect(resp).toHaveProperty("flagLen");
-  expect(resp).toHaveProperty("flagsValue");
-  expect(resp).toHaveProperty("flag_recovery");
-  expect(resp).toHaveProperty("flag_signed_mcu_code");
-  expect(resp).toHaveProperty("flag_onboarded");
-  expect(resp).toHaveProperty("flag_pin_validated");
-});
+  test("showAddressAndPubKey_Secp256k1", async () => {
+    const app = new OasisApp(transport);
 
-test("deviceInfo", async () => {
-  const transport = await TransportNodeHid.create(1000);
-  const app = new OasisApp(transport);
+    // Derivation path. First 3 items are automatically hardened!
+    const path = [44, 60, 0];
```

Comment:
Please also cover the case for the 2nd key `[44', 60', 0', 0, 1]`. The helper function in `helperV1.js` will probably need to be updated or a new one added, but please don't break backward compatibility.